docs/guides/ssr.md

```diff
@@ -111,6 +180,95 @@ function Posts() {
 
 As demonstrated, it's fine to prefetch some queries and let others fetch on the queryClient. This means you can control what content server renders or not by adding or removing `prefetchQuery` for a specific query.
 
+#### With experimental `app` directory
```

Comment:
I'd add a big `:::caution` under here that this is experimental and is subject to change awaiting _{{some link to a React-issue or RFC}}_